Gemfile

```diff
@@ -9,6 +9,7 @@ gem "bootstrap-kaminari-views"
 gem "bootstrap-sass", "~> 3.4.1"
 gem "config"
 gem "faker", "~> 2.1.2"
+gem "figaro"
```

Comment:
gem này để làm gì e ?

Reply:
gem Figaro, bạn dễ dàng cấu hình các biến. Quản lý toàn bộ biến môi trường : ở đây là username và password của tài khoản gmail